_inc/lib/admin-pages/class.jetpack-settings-page.php

```diff
@@ -55,6 +55,17 @@ function page_render() {
 			$version_notice
 		);
 
+		$rest_api_notice = str_replace(
+			'#HEADER_TEXT#',
+			esc_html( __( 'WordPress REST API is disabled', 'jetpack' ) ),
+			$rest_api_notice
+		);
+		$rest_api_notice = str_replace(
+			'#TEXT#',
+			esc_html( __( "Enable WordPress REST API to unlock Jetpack's full potential!", 'jetpack' ) ),
```

Comment:
That text is a notice displayed when user has REST API restricted and is redirected to old Settings page. BTW @zinigor you might want to use `esc_html__` instead of `__` and then `esc_html`.